test/unit/Reflection/ReflectionClassTest.php

```diff
@@ -2081,7 +2089,7 @@ public function testGetImmediateConstants(): void
         self::assertSame('ff', $reflectionConstants['F']->getValue());
     }
 
-    /** @return list<array{0: int, 1: int}> */
+    /** @return list<array{0: int-mask-of<CoreReflectionClassConstant::IS_*>, 1: int}> */
     public function getConstantsWithFilterDataProvider(): array
     {
         return [
```

Comment:
Hmm, PHPStan kinda disagreeing here?

Reply:
`ReflectionClassConstant::IS_FINAL` exists since 8.1 that's why the problem is on PHP 8.0. I'm not sure what's the best way how to solve it.

Reply:
Can we perhaps declare our own constant on the adapter, then use `ReflectionClass::IS_*`?

I think the constants aren't `final`

Reply:
Perfect👍 Done.